src/commonmark/en/content/user/data-administration.md

```diff
@@ -608,6 +610,21 @@ Some aspects of the data synchronization feature to be aware of:
     The validation won't fail for the user with this authority. The
     authority should be assigned to the user, on the central server,
     that will be used for synchronization job.
+    
+  - <div id="skip_changed_before"></div>In specific cases, **the initial 
+    synchronisation of all the data can be redundant.** Such situation can 
```

Comment:
I'm not sure "redundant" is exactly the right word here, given the examples that follow. I suggest to change this to "undesirable". e.g. (with a couple of additional tweaks):

In specific cases, **the initial synchronisation of all of the data can be undesirable**; for example, when a database on...